eve/main.yml

```diff
@@ -605,6 +664,116 @@ stages:
+          workdir: build/eve/workers/openstack-single-node-rhel/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Create ISO mountpoint in bootstrap node
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo mkdir -p /var/tmp/metalk8s
+          workdir: build/eve/workers/openstack-single-node-rhel/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Mount ISO in bootstrap node
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo mount -o loop /tmp/metalk8s.iso /var/tmp/metalk8s
+          workdir: build/eve/workers/openstack-single-node-rhel/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Start the bootstrap process
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo bash /var/tmp/metalk8s/bootstrap.sh --verbose
+          workdir: build/eve/workers/openstack-single-node-rhel/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Provision Prometheus and AlertManager storage
+          env:
+            SSH_CONFIG: >-
+              eve/workers/openstack-single-node-rhel/terraform/ssh_config
+          command: >
+            scp -F $SSH_CONFIG eve/create-volumes.sh bootstrap:/tmp/create-volumes.sh &&
+            ssh -F $SSH_CONFIG bootstrap
+            sudo env
+            PRODUCT_TXT=/var/tmp/metalk8s/product.txt
+            PRODUCT_MOUNT=/var/tmp/metalk8s
+            /tmp/create-volumes.sh
+          haltOnFailure: true
+      - ShellCommand:
+          name: Run fast tests
+          env:
+            SSH_CONFIG_FILE: "eve/workers/openstack-single-node-rhel/terraform/ssh_config"
+            ISO_MOUNTPOINT: "/var/tmp/metalk8s"
+            TEST_HOSTS_LIST: "bootstrap"
+            PYTEST_FILTERS: "post and ci and not multinodes and not slow"
+          command: >
+            tox -e tests -- -m "${PYTEST_FILTERS}"
+          haltOnFailure: true
+      - ShellCommand:
+          name: Generate logs using sosreport
+          command: >
+             ssh -F ssh_config bootstrap
+               "sudo sosreport --all-logs -o metalk8s -kmetalk8s.podlogs=True
+               -o containerd -kcontainerd.all=True -kcontainerd.logs=True
+               --batch --tmp-dir /var/tmp &&
+               sudo chown cloud-user:cloud-user /var/tmp/sosreport*"
+          workdir: build/eve/workers/openstack-single-node-rhel/terraform/
+          alwaysRun: true
+      - ShellCommand:
+          name: Collect sosreport from the bootstrap node
+          env:
+            SSH_CONFIG: >-
+              eve/workers/openstack-single-node-rhel/terraform/ssh_config
+          command: >
+            mkdir -p sosreport/sosreport/single-node-install-rhel &&
+            scp -F $SSH_CONFIG bootstrap:/var/tmp/sosreport-* \
+              sosreport/sosreport/single-node-install-rhel/bootstrap-sosreport.tar.xz
+          alwaysRun: true
```

Comment:
Could be worth factorizing with `multiple-nodes` equivalent steps IMO

Reply:
done